modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -2,33 +2,40 @@ package keeper
 
 import (
 	"bytes"
+	"context"
 	"encoding/hex"
 
 	wasmvm "github.com/CosmWasm/wasmvm/v2"
 
+	"cosmossdk.io/collections"
 	"cosmossdk.io/core/store"
 	errorsmod "cosmossdk.io/errors"
+	"cosmossdk.io/log"
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/ibcwasm"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
 	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v8/modules/core/exported"
 )
 
 // Keeper defines the 08-wasm keeper
 type Keeper struct {
 	// implements gRPC QueryServer interface
 	types.QueryServer
-
-	cdc codec.BinaryCodec
-
+	// vm contains an implementation of the WasmEngine
+	vm ibcwasm.WasmEngine
+	// state management
+	schema       collections.Schema
+	checksums    collections.KeySet[[]byte]
 	storeService store.KVStoreService
-
+	// handling queries
+	queryPlugins QueryPlugins
+	cdc          codec.BinaryCodec
 	clientKeeper types.ClientKeeper
-
-	authority string
+	authority    string
```

Comment:
Just for readability: maybe add an empty line to separate these? Or move them to the top?